documentation/api/introduction.rst

```diff
@@ -179,27 +136,26 @@ Throughout this document, keys are written in singular if a single value is list
 
 The API, however, does not distinguish between singular and plural key notation.
 
-Connections and entity addresses
+Sensors and entity addresses
```

Comment:
This gives the impression that this is something you have to know to use the API. But at this point, you don't have to know it. I suggest giving a bit more information about the context in which this info might be useful.

All in all, the whole "Notation" section in this API introduction could use a helicopter perspective on how to put the content into perspective, but that is maybe another PR.